Makefile

```diff
@@ -61,7 +61,10 @@ test-cpp:
 	cd build && ctest --no-compress-output -T Test -C RelWithDebInfo --output-on-failure
 	xsltproc .github/kokoro/ctest2junit.xsl build/Testing/*/Test.xml > build/cpp_test_results.xml
 
-.PHONY: test test-py test-cpp
+test-tools:
+	$(MAKE) -f Makefile.tools_tests
+
+.PHONY: test test-py test-cpp test-tools
```

Comment:
Should `test-tools` be added to the `test` pseudo target?

Reply:
Done